pilot/cmd/pilot-agent/main.go

```diff
@@ -105,6 +108,37 @@ var (
 		"The proxy configuration. This will be set by the injection - gateways will use file mounts.",
 	).Get()
 
+	caProviderEnv = env.RegisterStringVar("CA_PROVIDER", "Citadel", "name of authentication provider").Get()
+	// TODO: default to same as discovery address
+	caEndpointEnv = env.RegisterStringVar("CA_ADDR", "", "Address of the spiffee certificate provider. Defaults to discoveryAddress").Get()
+
+	// TODO: this is a horribly named env, it's really TOKEN_EXCHANGE_PLUGINS - but to avoid breaking
+	// it's left unchanged. It may not be needed because we autodetect.
+	pluginNamesEnv = env.RegisterStringVar("PLUGINS", "", "Token exchange plugins").Get()
+
+	// This is also disabled by presence of the SDS socket directory
+	enableGatewaySDSEnv = env.RegisterBoolVar("ENABLE_GATEWAY_SDS", false,
```

Comment:
Accidental - a previous PR that caused lots of merge conflicts renamed all 'ingress gateways' to 'gateway' - since egress will
also use this. But the env must stay the same, fixing it.